hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -474,9 +459,16 @@ public Hashtable<String, String> getPathStatus(final Path path,
               path);
 
       final Hashtable<String, String> parsedXmsProperties;
+      final String relativePath = getRelativePath(path);
+      final EncryptionAdapter encryptionAdapter
+          = createEncryptionAdapterFromServerStoreContext(relativePath,
+          tracingContext);
       final AbfsRestOperation op = client
-          .getPathStatus(getRelativePath(path), true, tracingContext);
+          .getPathStatus(relativePath, true, tracingContext, encryptionAdapter);
```

Comment:
This is in-line with https://github.com/apache/hadoop/pull/3440#pullrequestreview-1133031625, where we don't want AbfsClient to have logic to get an encryptionAdapter: https://github.com/sumangala-patki/hadoop/blob/HADOOP-17912/hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java#L251-L262